radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
     self.imageArray = sitk.GetArrayFromImage(self.inputImage)
-    self.maskArray = (sitk.GetArrayFromImage(self.inputMask) == self.label).astype('int')
+    self.maskArray = (sitk.GetArrayFromImage(self.inputMask) == self.label)  # boolean array
 
-    self.matrix = self.imageArray.astype('float')
-    self.matrixCoordinates = numpy.where(self.maskArray != 0)
+    self.ROICoordinates = numpy.where(self.maskArray != 0)
```

Comment:
Mybe `nonzeroVoxelCoordinates` or `labeledVoxelCoordinates` is a more intuitive name? When I think about ROI, I think either a box, or a segmented region, and it is not immediately clear what `ROICoordinates` means.